hdf5-blosc2/src/blosc2_filter.c

```diff
@@ -214,14 +214,14 @@ size_t blosc2_filter_function(unsigned flags, size_t cd_nelmts,
       goto failed;
     }
 
-    status = blosc2_schunk_append_buffer(schunk, *buf, nbytes);
+    status = (int) blosc2_schunk_append_buffer(schunk, *buf, (int32_t) nbytes);
```

Comment:
To better catch error conditions, I think it is better to declare `status` as int64_t